### ISSUE TITLE
TypeError: this.updater.enqueueCallback is not a function

### ISSUE THREAD

**Issue by roycebranning** [twisty/formsy-react-components#118]
Uncaught TypeError: this.updater.enqueueCallback is not a function
    at WrappedComponent.ReactComponent.setState (ReactBaseClasses.js?1bef:64)
    at FormsyForm.validate (index.js?5af9:340)
    at Object.attachToForm (index.js?5af9:499)
    at configure (HOC.js?9583:127)
    at WrappedComponent.componentWillMount (HOC.js?9583:135)
    at callComponentWillMount (react-dom.development.js?cada:9777)
    at mountClassInstance (react-dom.development.js?cada:9834)
    at updateClassComponent (react-dom.development.js?cada:10216)
    at beginWork (react-dom.development.js?cada:10605)
    at performUnitOfWork (react-dom.development.js?cada:12573)

**Comment by twisty** [comment]
Not enough info to help you with this. If you haven't sorted this yet please open a new issue with steps to reproduce / code samples / etc.

FYI first result on Google for "`TypeError: this.updater.enqueueCallback is not a function`" is https://github.com/facebook/react/issues/10320. Are you sure you aren't somehow including React 15 and React 16?

We're waiting for [`formsy-react` to support React 16](https://github.com/christianalfoni/formsy-react/issues/481) before we can state that we're compatible with React 16.

**Comment by vchaudhari** [comment]
**I am facing following issue can anybody help me out
The issue occurs when I am trying to use callback function for react-x-editable component.**

Uncaught TypeError: this.updater.enqueueCallback is not a function
    at t.r.setState (editable.js:28)
    at t.n.onSubmit (editable.js:28)
    at HTMLUnknownElement.callCallback (react-dom.development.js:100)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:138)
    at Object.invokeGuardedCallback (react-dom.development.js:187)
    at Object.invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:201)
    at executeDispatch (react-dom.development.js:461)
    at executeDispatchesInOrder (react-dom.development.js:483)
    at executeDispatchesAndRelease (react-dom.development.js:581)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:592)
    at forEachAccumulated (react-dom.development.js:562)
    at runEventsInBatch (react-dom.development.js:723)
    at runExtractedEventsInBatch (react-dom.development.js:732)
    at handleTopLevel (react-dom.development.js:4472)
    at batchedUpdates$1 (react-dom.development.js:16537)
    at batchedUpdates (react-dom.development.js:2131)
    at dispatchEvent (react-dom.development.js:4551)
    at interactiveUpdates$1 (react-dom.development.js:16592)
    at interactiveUpdates (react-dom.development.js:2150)
    at dispatchInteractiveEvent (react-dom.development.js:4528)
r.setState @ editable.js:28
n.onSubmit @ editable.js:28
callCallback @ react-dom.development.js:100
invokeGuardedCallbackDev @ react-dom.development.js:138
invokeGuardedCallback @ react-dom.development.js:187
invokeGuardedCallbackAndCatchFirstError @ react-dom.development.js:201
executeDispatch @ react-dom.development.js:461
executeDispatchesInOrder @ react-dom.development.js:483
executeDispatchesAndRelease @ react-dom.development.js:581
executeDispatchesAndReleaseTopLevel @ react-dom.development.js:592
forEachAccumulated @ react-dom.development.js:562
runEventsInBatch @ react-dom.development.js:723
runExtractedEventsInBatch @ react-dom.development.js:732
handleTopLevel @ react-dom.development.js:4472
batchedUpdates$1 @ react-dom.development.js:16537
batchedUpdates @ react-dom.development.js:2131
dispatchEvent @ react-dom.development.js:4551
interactiveUpdates$1 @ react-dom.development.js:16592
interactiveUpdates @ react-dom.development.js:2150
dispatchInteractiveEvent @ react-dom.development.js:4528
